### DISCUSSION TITLE
JWMMF: New name

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#316] [General]
JWMMF has changed so much since the early days. It used to be a simple downloader and  not much more, borne out of a .bat file that I had made for myself to help me manage our media at meetings. 

Now, it's grown into a full fledged solution to handle selective downloading of needed meeting media from JW.org, syncing with other users in the same congregation through WebDAV, media picture and video conversion, presenting media on external displays, and even integrating with broadcasting software!

Recently, some have also expressed concern about the fact that the name of the app contains the letters "JW". While this isn't problematic from a legal standpoint, nor does the app purport to be an official app in any way, it seems that some hesitate to use the app simply due to its name.

Finally, JWMMF is really hard to say. 

For all these reasons, I think the time has come to change JWMMF's name.

However, it turns out that naming things is hard, so I don't have much inspiration. The only thing I came up with so far is **Cammm** (congregation automated meeting media manager). 

### Do any of you have any ideas for app names that would encompass the whole gist of what JWMMF can do?

___

Some key words I came up with, to help think of names:
- Meeting
- Media
- Assistant
- Manager
- Updater
- Upload
- Download
- Sync
- Presentation
- Display
- Theocratic
- Easy
- Converter
- Hub
- Congregation

**Comment by Uilsonjw** [comment]
How about DPMCong (Download and play media on congregation)
or
DPMMeet (Download and play media on meetings)
The second sound better for me.

**Reply by LucasMagalhaesST** [reply]
The name sounds very interesting, easy to remember.

**Reply by Jonathas1987** [reply]
I liked the second suggestion: **DPMMeet (Download and play media on meetings)**

**Comment by joaochris** [comment]
CongPMedia
MPMedia

Or just, not to change the program name drastically:
Congregation Meeting Media Fetcher
Meeting Media Fetcher

**Comment by LucasMagalhaesST** [comment]
As we are talking about software that will help us in meetings, my suggestion:

AM (Meeting Assistant)

**Comment by e-Feitosa** [comment]
- [ ] **OneClick Meeting**

or

- [ ] Easy Meeting (EM)
- [ ] Assistant Meeting (AM)
- [ ] Assistant Zoom (AZ)
- [ ] Hub Meeting (HM)
- [ ] Meeting Media Fetcher (M²F)

**Reply by brunolaj90** [reply]
I Liked "Easy Meeting". Summarizes the purpose of the program.

**Comment by brunolaj90** [comment]
Meeting Media Fetcher (MMF)
or
Meeting Media Player (MMP)

**Reply by brunolaj90** [reply]
I also liked "Easy Meeting" and "Smart Meeting"

**Comment by SandroCODTEC** [comment]
Yes, naming a system is often more difficult than naming a child. I go through this every time I start a new project.

In addition to being pronunciation, he must have it easy in other languages. It would also be a utopia encompassing all aspects of the system does. Thus, it is reasonable to be content with the essence of the project.

_So I suggest:_
**Smart Media Projector**
We can just _pronounce_ **Smart Media** or **SMP**.

**Comment by wilianbritto** [comment]
Sugestão: Media Center for meetings -MCM
Central de mídias para reuniões CMR
Levando em conta todas as funções do programa esse no meu ver é muito adequado!

**Reply by Jonathas1987** [reply]
Write in English, my dear Goga! 

Greetings!

**Reply by wilianbritto** [reply]
Media Center for meetings

Saudades goguito

**Reply by wilianbritto** [reply]
Coloquei mais num vai

**Reply by wilianbritto** [reply]
`media center meetings`

**Comment by aleraniere** [comment]
MPM - Media Pool for Meetings

**Reply by wilianbritto** [reply]
Pool ou pull

**Reply by aleraniere** [reply]
https://www.google.com/search?q=media+pool

> Pool ou pull

**Comment by sircharlo** [comment]
What about MMM or M³?
**M**eeting **M**edia **M**anager.
In speech, could be called "three-em", "em-three", "em-em-em", or "triple-em".

**Reply by joaochris** [reply]
I really liked this one, so just take and put words, it doesn't deviate much from the original name.

**Reply by e-Feitosa** [reply]
M³

**Reply by brunolaj90** [reply]
> I really liked this one, so just take and put words, it doesn't deviate much from the original name.

Exactly. I liked too.

**Reply by filipemello** [reply]
I liked it!

**Reply by Jonathas1987** [reply]
Very good! Top!

**Reply by sircharlo** [reply]
Alright, I think this is what I'll go with finally. Simple, clear, concise, easy to pronounce, and controversy-free.

**Reply by stepstra** [reply]
Yes, M³

**Comment by mjacobus** [comment]
Maybe a pun or a gist with/of a bible teaching. Scriptures that show Jesus or Jehovah using illustrations.

- https://www.jw.org/en/library/books/reading-and-teaching/8-illustrations-that-teach/
- https://www.jw.org/en/library/books/reading-and-teaching/9-appropriate-use-of-visual-aids/
- Kingdom Vision
- Kingdom Viewer
- Imagine
- Envision
- Something with Hebrews 11:1 where it speaks of realities not [yet] seen.

**Reply by mjacobus** [reply]
- [Imagine the time](https://www.jw.org/en/library/music-songs/original-songs/imagine-the-time/).

**Comment by riemsesy** [comment]
JW M³ is perfect :-)
